docs/HowToDebug.md

```diff
@@ -1,4 +1,80 @@
 **How to Debug in NNI**
 ===
 
-*Coming soon*
+## Overview
+
+There are three parts that might have logs in NNI. They are nnimanager, dispatcher and trial. Here we will introduce them succinctly. More information please refer to [Overview](Overview.md).
+
+- **nnictl**: NNI controller is the nni command line tool that is used to started NNI.

```

Comment:
`NNI controller` -> `nnictl`
`command line` -> `command-line`
`to started NNI` -> `to manage experiments (e.g., start an experiment)`